### DISCUSSION TITLE
有想一起去芭莱大厦寻宝的吗？

### DISCUSSION THREAD

**Discussion by Guessme404** [share121/inter-knot#3022] [其他]
现招募志同道合的伙伴，一起发家致富！（有意私聊~）
![b89b47c56e804d3841a99ead323985c](https://github.com/user-attachments/assets/d804d73c-ab72-4ab0-adc8-2c13ad8d4e00)

**Comment by Ruanachan** [comment]
我有朋友三天前去过那里，现在还在家里窝着说什么都不出来呢，因应该是被吓惨了，我劝楼主还是放弃的好<img src="https://bbs-static.miyoushe.com/static/2023/10/07/fdd3cfc769e04bddf0a6818531135160_8889794404812794499.png" alt="蒂亚拉_不乐" width="75" >

**Comment by fashibuding1214** [comment]
怎么约，细说
